frontend/src/components/labwork/step/LabworkStepOverview.tsx

```diff
@@ -56,7 +56,7 @@ const LabworkStepOverview = ({step, refreshing, stepSamples, columns, filterDefi
 
   useEffect(() => {
     dispatch(getLabworkStepSummary(step.id, activeGrouping.key, {}))
-  }, [])  // Fetches the initial labwork step summary
+  }, [activeGrouping.key, dispatch, step.id])  // Fetches the initial labwork step summary
```

Comment:
i think this was meant to fire only once ... some other use effect triggers to refresh the grouping and step ... I don't recall where but at some place there was an infinite loop related to circular dependencies ...